manifest.json

```diff
@@ -1,5 +1,87 @@
 {
   "name": "Automated merging",
   "description": "Automatically merge pull-requests.",
-  "ubiquity:listeners": ["push", "issue_comment.created"]
-}
+  "ubiquity:listeners": [
+    "push",
+    "issue_comment.created"
+  ],
+  "configuration": {
```

Comment:
I'm debating if we should have a different manifest file for this because the original manifest is super simple and clean. Any strong case for either?

Reply:
Advantages are that you have the full information about the plugin in one place, with a single endpoint. Also these files are cached within the kernel during the run so it greatly reduces networks calls (we are limited to 50 in the kernel and using the same file would divide calls by 2). Configuration is a valid manifest entry as well.